### DISCUSSION TITLE
About Android sensors

### DISCUSSION THREAD

**Discussion by tongzifang** [getsentry/sentry-java#2393] [Ideas]
Collection of sensor information may violate the Privacy Policy,
This behavior is controlled by the system-event breadcrumbs,
Can it be configured separately so that other system events are not affected?
![image](https://user-images.githubusercontent.com/18605773/204205945-eb3c884e-3b3c-4d6f-ad20-4eab4e6e1ef7.png)

**Comment by romtsn** [comment]
You can remove it already now by doing `Sentry.getCurrentHub().getOptions().getIntegrations().remove(<integration instance>)`, though it's not public API. We'll discuss internally whether we should remove it from default integrations altogether
